src/streaming/utils/Capabilities.js

```diff
@@ -138,6 +138,11 @@ function Capabilities() {
             return Promise.resolve();
         }
 
+        const enhancementCodecs = settings.get().streaming.enhancement.codecs;
+        if (settings.get().streaming.enhancement.enabled && basicConfiguration.codec.includes('video') && enhancementCodecs.some(cdc => basicConfiguration.codec.includes(cdc))) {
```

Comment:
* Use `Constants` for type: `Constants.Video`
* Do we need to limit this to `video`?